db/schema.rb

```diff
@@ -10,22 +10,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2019_11_22_175335) do
+ActiveRecord::Schema.define(version: 2021_02_01_100504) do
 
   create_table "browsers", force: :cascade do |t|
-    t.string "name"
+    t.string "name", limit: 255
+    t.datetime "created_at", precision: 6, null: false
+    t.datetime "updated_at", precision: 6, null: false
     t.index ["name"], name: "index_browsers_on_name"
   end
 
   create_table "campaigns", force: :cascade do |t|
-    t.string "utm_source", limit: 50
-    t.string "utm_medium", limit: 50
-    t.string "utm_campaign", limit: 50
-    t.string "utm_content", limit: 50
-    t.string "utm_term", limit: 50
+    t.string "utm_source", limit: 256
```

Comment:
I think this should be 255 to be varchar unless you wanted them to be text ?

Reply:
I checked the migration file they are still specified as strings so maybe this was osme migration fluke